packages/advanced-gallery/src/attributes.json

```diff
@@ -45,5 +45,13 @@
 	"defaultsGenerated": {
 		"type": "boolean",
 		"default": false
+	},
+	"overlayFilterStyle": {
+		"type": "string",
+		"default": "unitone"
+	},
+	"overlayFilterStrength": {
+		"type": "number",
+		"default": 100
```

Comment:
These attributes are also added in the `packages/block-editor/src/filters/with-overlay-filter-strength-controls/attributes.json` file.